assets/airports/othh.json

```diff
@@ -189,151 +1798,604 @@
+            ]
+        },
+        "ALVEN1C": {
+            "icao": "ALVEN1C",
+            "name": "Alven One Charlie",
+            "rwy": {
+                "OTHH16R": []
+            },
+            "body":[["OBVER", "A40|S250-"], ["SODER", "S250-"], "MUSAN", "ENELI", ["TUKEN", "A40"], ["ALKAG", "A50+"], ["DEBIL", "A60"], ["ALVEN", "A60"]],
+            "exitPoints": {
+                "ALVEN": []
+            },
+            "draw": [
+                ["_RWY34L", "OBVER", "SODER", "MUSAN", "ENELI", "TUKEN", "ALKAG", "DEBIL", "ALVEN*"]
+            ]
+        },
+        "ALVEN1E": {
+            "icao": "ALVEN1E",
+            "name": "Alven One Echo",
+            "rwy": {
+                "OTHH34R": []
+            },
+            "body":[["SOKEN", "A25+"], ["DEMBO", "A40"], ["GIBIN", "A50+"], ["ITSET", "A70"], "ALVEN"],
+            "exitPoints": {
+                "ALVEN": []
+            },
+            "draw": [
+                ["_RWY16L", "SOKEN", "DEMBO", "GIBIN", "ITSET", "ALVEN*"]
+            ]
+        },
+        "ALVEN1M": {
+            "icao": "ALVEN1M",
+            "name": "Alven One Mike",
+            "rwy": {
+                "OTHH16L": []
+            },
+            "body":["TORKO", ["PASAV", "A40+"], ["DEBOM", "A50+"], ["PEGEM", "A60+"], ["GOBLU", "A70"], "SITAL", "ALVEN"],
+            "exitPoints": {
+                "ALVEN": []
+            },
+            "draw": [
+                ["_RWY34R", "TORKO", "PASAV", "DEBOM", "PEGEM", "GOBLU", "SITAL", "ALVEN*"]
+            ]
+        },
+        "ALVEN1W": {
+            "icao": "ALVEN1W",
+            "name": "Alven One Whiskey",
+            "rwy": {
+                "OTHH34L": []
+            },
+            "body":[["GETAM", "A25+"], "ENELI", ["KOBEN", "A30"], "DEBIL", ["VATEK", "A30"], ["NABKI", "A40"], ["ALVEN", "A60-"]],
+            "exitPoints": {
+                "ALVEN": []
+            },
+            "draw": [
+                ["_RWY16R", "GETAM", "ENELI", "KOBEN", "DEBIL", "VATEK", "NABKI", "ALVEN*"]
+            ]
+        },
+        "BATHA1C": {
+            "icao": "BATHA1C",
+            "name": "Batha One Charlie",
+            "rwy": {
+                "OTHH16R": []
+            },
+            "body":[["OBVER", "A40+"], "LADEM", ["EMEXA", "A50+"], ["SOLAL", "A60+"], ["TONBI", "A70"], "DENSI", "BAT"],
+            "exitPoints": {
+                "BAT": []
+            },
+            "draw": [
+                ["_RWY34L", "OBVER", "LADEM", "EMEXA", "SOLAL", "TONBI", "DENSI", "BAT*"]
+            ]
+        },
+        "BATHA1E": {
+            "icao": "BATHA1E",
+            "name": "Batha One Echo",
+            "rwy": {
+                "OTHH34R": []
+            },
+            "body":[["ELEDA", "S250-"], ["^MEBTI", "S250-"], ["BOSUP", "A30+"], ["MUXED", "A40"], ["LADEM", "A40"], ["EMEXA", "A50+"], ["SOLAL", "A60"], "DENSI", ["BAT", "A60"]],
```

Comment:
`MEBTI` is not a fly-over fix

Reply:
I know but without the fly-over fix the routing looks really odd:
With fly-over fix:
![ss](https://i.imgur.com/6lzC8h0.png)    
Without fly-over fix:
![ss](https://i.imgur.com/9Fl9N7v.png)